shell.nix

```diff
@@ -0,0 +1,10 @@
+{ tdds-src ? fetchGit {
+    url = https://github.com/makerdao/testchain-dss-deployment-scripts.git;
+    ref = "master";
```

Comment:
This is also not reproducible though. Don't you think we should be pinning one commit?

Reply:
You are right, it's not reproducible in the sense that this repo might get different envs from `tdds`. But the env it gets should be consistent within `tdds` and this was an example to recreate how `setup.sh` gets the latest `tdds` repo regardless.

Depends on the use case, could definitely set a fixed commit.

Reply:
If the goal is to get the latest, then this is fine. Maybe we should document this somewhere.